### PR TITLE
uploading database research doc

### PR DESCRIPTION
**Pull request by SamrajMahadev**
This is the research documents on different types of database, different database pros and cons, comparison on various  database platform. The reason for deciding why the team choose to use MySQL database platform.

### CODE REVIEW

File: foodremedy-api/database-research.md
Comment (deasta) [inline foodremedy-api/database-research.md:1]: Consider changing filename to specify what you are researching such as 'Relational vs. Non-Relational database research.md'

Comment (deasta) [reply foodremedy-api/database-research.md:1]: also quickly add an introduction explaining what this document is.

**Review by deasta — CHANGES_REQUESTED** [review]

File: foodremedy-api/database-research.md
Comment (SamrajMahadev) [reply foodremedy-api/database-research.md:1]: thank you for reviewing, i'll make the changes and updated the files..

Comment (abinabra) [reply foodremedy-api/database-research.md:1]: The changes look good and the introduction and the conclusion looks good. I will resolve this now.

**Pull request merged**